### DISCUSSION TITLE
Livewire sessions table not found

### DISCUSSION THREAD

**Discussion by gofish543** [livewire/livewire#7832] [1. Help] [answered]
I cloned the livewire repository and tried to create and run a test case but keep getting a “sessions table not defined” from the testbench connection. What is going on

**Comment by calebporzio** [comment]
Hmmm I've never heard of that before. Testbench is probably using a db session driver or something like that. It should handle all that on it's own so that's confusing why you would get that - maybe like not having sqlite extension or something installed? idk

**Reply by ju5t** [reply, marked as answer]
I've had this.

Adding this below the `CACHE_DRIVER` in phpunit.xml.dist helped for me:

```xml
<env name="SESSION_DRIVER" value="file"/>
```
